ramls/types/resources/resourceIncludedItemAttributes.json

```diff
@@ -0,0 +1,178 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Resource object schema for included item attributes",
+  "description": "Resource object schema for included item attributes",
+  "javaType": "org.folio.rest.jaxrs.model.ResourceIncludedItemAttributes",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
```

Comment:
I am a little curious about this schema. So, included can have "providers", "packages" or "titles". Did we end up including properties of all 3 in here?

Reply:
Yes, it's required because in case we include all of them "included" will contain "packages","providers","titles" and each of them has different properties.

![included](https://user-images.githubusercontent.com/44401279/48476956-0faac680-e809-11e8-8e20-6756772c8cc0.PNG)

Reply:
Ok, sounds good.